Libraries/Components/ScrollView/ScrollView.js

```diff
@@ -682,10 +682,10 @@ if (Platform.OS === 'android') {
       sendMomentumEvents: true,
     }
   };
-  AndroidScrollView = requireNativeComponent('RCTScrollView', ScrollView, nativeOnlyProps);
+  AndroidScrollView = requireNativeComponent('RCTScrollView', (ScrollView: ReactClass<*>), nativeOnlyProps);
```

Comment:
Maybe because of `createClass`? Bummer. Would be nice to kill that mixin and switch this to es6 at some point...